Userland/Applications/PixelPaint/Image.cpp

```diff
@@ -51,13 +51,13 @@ void Image::paint_into(GUI::Painter& painter, Gfx::IntRect const& dest_rect) con
     }
 }
 
-ErrorOr<NonnullRefPtr<Gfx::Bitmap>> Image::try_decode_bitmap(ReadonlyBytes bitmap_data)
+ErrorOr<NonnullRefPtr<Gfx::Bitmap>> Image::try_decode_bitmap(StringView path, ReadonlyBytes bitmap_data)
 {
     // Spawn a new ImageDecoder service process and connect to it.
     auto client = TRY(ImageDecoderClient::Client::try_create());
 
     // FIXME: Find a way to avoid the memory copying here.
-    auto maybe_decoded_image = client->decode_image(bitmap_data);
+    auto maybe_decoded_image = client->decode_image_with_known_path(path, bitmap_data);
```

Comment:
Please don't introduce any more uses of null strings :thousandyakstare:

Reply:
Sure, have some empty strings instead!